akka-operator/templates/050-gcp-application-cr.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.provider.name }}
+{{- if eq .Values.provider.name "gcp" }}
+{{- if eq (toString .Values.gcp.deployer) "false" }}
+apiVersion: app.k8s.io/v1beta1
```

Comment:
This is the `application` created for the Marketplace to display in the UI.
ref: https://github.com/lightbend/akka-cloud-platform-gcp/blob/main/deployer/base-application.yaml

Basically, the Marketplace references installed applications by a CR like this and displayes the information in the console (the UI).